h2o-persist-s3/build.gradle

```diff
@@ -10,7 +10,14 @@ description = "H2O Persist S3"
 
 dependencies {
   compile project(":h2o-core")
-  compile 'com.amazonaws:aws-java-sdk-s3:1.10.77'
+  // Exclude all fasterxml dependencies from aws-java-sdk and replace them with newer ones with latest fixes
+  compile ('com.amazonaws:aws-java-sdk-s3:1.11.818'){
+    exclude group: "com.fasterxml.jackson.core"
+  }
+  runtime 'com.fasterxml.jackson.core:jackson-core:2.11.1'
```

Comment:
core and databind are already bundled by `'com.google.cloud:google-cloud-storage:1.111.1'` from another repository, yet I'm including them here to make sure this will always work no matter if GCS is included or not.